app/helpers/uuid_helper.py

```diff
@@ -1,7 +1,7 @@
 from uuid import UUID
 
 
-def is_valid_uuid(uuid_string, version=4):
+def is_valid_uuid(uuid_string: str, version: int = 4) -> bool:
```

Comment:
Given this is for typing, I don't think that should be done here.

Reply:
I think I have misunderstood you. Did you need literally a constant, i.e like `DEFAULT_VERSION` etc?.

Reply:
[`bc7b703` (#822)](https://github.com/ONSdigital/eq-questionnaire-runner/pull/822/commits/bc7b70375fec18cc7291543383d598e1121a5d64)

Reply:
What i meant was , no need to pass args. I don't think there will be an instance where you will (call )validate uuid string with different versions.
```
def is_valid_uuid(uuid_string: str) -> bool:
    try:
        UUID(uuid_string, version=DEFAULT_UUID_VERSION)
    except ValueError:
```

Reply:
Not really type hint specific, but changed while I'm here. Decided to remove the constant since it is only used one, and renamed the method to be specific.